Localizations/webroot/localization_webroot.pot

```diff
@@ -28,6 +28,12 @@ msgstr ""
 msgid "...loading..."
 msgstr ""
 
+#: index.html:294
+msgid ""
+"<p>Connection to server was rejected due to invalid authentication. Reload browser window to reconnect to server properly. </p>\n"
+"                <p>If this problem persist open this page from the TrayIcon instead.</p>"
```

Comment:
I do not like the indents and `\n` here since it affects the localization UX on Transifex, but it looks it should be taken care of by our script to update pot files.